### ISSUE TITLE
About activity showing white background 

### ISSUE THREAD

**Issue by GauravChaddha1996** [nickbutcher/plaid#161]
First of all, I really appreciate this app. Thank you for making this. Now, my use case is to make an about-us section. So I thought to make it like the plaid app. I was able to replicate the about is view pager by understanding the plaid about activity code. But like in the plaid app about section, when we try to swipe any page up or down, we can see the home activity behind our page(Easily seen on the second page). But in my project, I see white background instead of the home activity(Even after keeping views the same as plaid app) . I have also kept the styles same and yet I am not able to replicate that effect. Does anyone have any idea why this is happening?

**Comment by GauravChaddha1996** [comment]
Solved it. It was my own stupidity for 3 hours, I was setting the fragment to null by mistake before starting the about activity.

**Comment by nickbutcher** [comment]
Glad you solved it and that you find Plaid useful 😁

**Comment by aseef17** [comment]
I have the same issue , can you let me know how you solved it ?

**Comment by nickbutcher** [comment]
You likely have a view with a white background in your hierarchy or window background. Use hierarchy viewer to investigate.

**Comment by GauravChaddha1996** [comment]
@nickbutcher is right. In my case the flow was like this - I had an activity running (Let's call it Activity1 It had a frame layout holding many fragments) and then I wanted to launch the about activity(Activity2) having a transparent background where I can see my Activity1. The thing I was doing wrong was setting a null fragment to my frame layout leading the Activity1 to show it's default background - White. I solved it by not changing the loaded fragment in Activity1 when I needed to launch Activity2.
